presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -577,7 +578,7 @@ public ExchangeBuilder addInputsSet(List<Symbol> inputs)
 
         protected ExchangeNode build()
         {
-            return new ExchangeNode(idAllocator.getNextId(), type, scope, partitioningScheme, sources, inputs);
+            return new ExchangeNode(idAllocator.getNextId(), type, scope, partitioningScheme, sources, inputs, Optional.ofNullable(orderingScheme));
```

Comment:
Changes to `PlanBuilder` in this commit are incomplete and are not used. I think these changes belong to later commit `Add support for OrderingScheme in PushProjectionThroughExchange`.